app/js/components/answers/Index.react.js

```diff
@@ -48,15 +48,17 @@ class AllAnswers extends React.Component {
 
    sortAnswers(answers_obj){
     let answers_arr = [], accepted_answer;
+
+    // converted the object into an array so we can sort
     for (var key in answers_obj) {
-      answers_obj[key].accepted 
+      answers_obj[key].accepted
```

Comment:
Hmmm. . . rails developers would beg to differ. It's our habit. Throughout the app, I've been using camelCase for function names, and snake_case for variables. 
If we're to change, it would have to be site wide. Can you propose this in next meeting?